### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by falsandtru**
Remove mocha-phantomjs and bower :hocho:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /node_modules
-/bower_components
+/npm-debug.log
+/espowered_tests
 /.idea
 /TODO.txt
-/espowered_tests
```
Comment (twada) [inline .gitignore]: Please avoid reordering lines to not to generate unnecessary diffs.

File: test/fixture/amd.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿<script type="text/javascript" src="/base/node_modules/requirejs/require.js"></script>
```
Comment (twada) [inline test/fixture/amd.html]: Where the `/base` come from? (Just a question)

File: test/tobe_instrumented/assertion.js [hunk 3]
```diff
@@ -27,15 +20,13 @@ describe('power-assert message', function () {
         };
     });
 
-
```
Comment (twada) [inline test/tobe_instrumented/assertion.js]: Please avoid deleting lines to not to generate unnecessary diffs.

File: .gitignore (hunk 1 above)
Comment (falsandtru) [inline .gitignore]: All right!

**Comment by twada** [comment]
@falsandtru Great pull-req!
I'll merge this once you apply my comments.

File: test/fixture/amd.html (hunk 2 above)
Comment (falsandtru) [inline test/fixture/amd.html]: See http://karma-runner.github.io/0.13/config/files.html

Comment (twada) [inline test/fixture/amd.html]: Thanks! That clears things up a lot.

**Comment by falsandtru** [comment]
Fixed. Can I squash these commits, or you'll do that by merging with squash?

**Comment by twada** [comment]
@falsandtru Thanks!! I'll squash and merge this one with new `Squash and merge` feature.

**Comment by falsandtru** [comment]
All right, thanks for merging!

**Comment by twada** [comment]
@falsandtru Refactored karma-based test suite is excellent. Thanks!

**Comment by falsandtru** [comment]
:+1:

**Pull request merged**